.github/workflows/release.yml

```diff
@@ -0,0 +1,281 @@
+      image: ${{ matrix.image }}
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - target: aarch64-debian-bullseye
+            platform: linux_aarch64-debian-bullseye
+            image: debian:bullseye
+            runs_on: buildjet-8vcpu-ubuntu-2204-arm
+          - target: x86_64-debian-bullseye
+            platform: linux_x86_64-debian-bullseye
+            image: debian:bullseye
+            runs_on: buildjet-8vcpu-ubuntu-2204
+          - target: aarch64-debian-bookworm
+            platform: linux_aarch64-debian-bookworm
+            image: debian:bookworm
+            runs_on: buildjet-8vcpu-ubuntu-2204-arm
+          - target: x86_64-debian-bookworm
+            platform: linux_x86_64-debian-bookworm
+            image: debian:bookworm
+            runs_on: buildjet-8vcpu-ubuntu-2204
+
+    steps:
+      - name: Checkout Code
+        uses: actions/checkout@v3
+        with:
+          ref: ${{ needs.prepare.outputs.sha }}
+
+      - name: Install dependencies
+        run: |
+          apt-get update
+          apt-get -y dist-upgrade
+          DEBIAN_FRONTEND=noninteractive apt-get -y --no-install-recommends install \
+              build-essential \
+              ca-certificates \
+              cmake \
+              curl \
+              doxygen \
+              g++ \
+              gdb \
+              git \
+              gnupg \
+              gpg \
+              less \
+              libabsl-dev \
+              libboost-all-dev \
+              libc-ares-dev \
+              libgrpc++-dev \
+              libprotobuf-dev \
+              libre2-dev \
+              libssl-dev \
+              ninja-build \
+              pkg-config \
+              protobuf-compiler-grpc \
+              software-properties-common \
+              sudo \
+              wget \
+              zlib1g-dev
+
+          apt-add-repository -y 'deb http://deb.debian.org/debian bullseye-backports main'
+
+          apt-get update
+          apt-get -y --no-install-recommends install -t clang-15 clang-tidy-15
+          apt-get -y install cmake
+
+          apt-get -y --no-install-recommends install -t bullseye-backports cmake
+      - name: Setup build directory
+        run: mkdir builds
+
+      - name: Build
+        shell: bash
+        run: |
+          mkdir build
+          /usr/bin/cmake -S . -B build -G Ninja
+          /usr/bin/cmake --build build --target all install -- -v -j 3
+
+      - name: Copy
+        run: |
+          /usr/bin/cmake --install build --prefix builds/libviamsdk-${{ matrix.platform }}
+
+      - name: Upload artifacts
+        uses: actions/upload-artifact@v3
+        with:
+          name: builds
+          path: builds
+
+  release:
+    needs: [prepare, build_macos, build_linux_debian, build_linux_ubuntu_jammy]
+    if: github.repository_owner == 'viamrobotics'
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/download-artifact@v3
+
+      - name: Release
```

Comment:
Is there somewhere I can see the results of this step, or is that not really possible until this merges?

Reply:
Not possible at this time unfortunately, but the job is identical to that used by `rust-utils` so I think we should be able to analogize from the releases in that repo (though replace specific binaries with the `viam-cpp-sdk` directories).